templates/page.tpl.php

```diff
@@ -87,18 +87,15 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
-    <div class="wrapper">
-      <section class="footer-top">
-        <nav>
-          <?php print render($page['footer']); ?>
-        </nav>
-      </section>
-    </div>
-  </section>
-  <section class="page-bottom">
-    <div class="wrapper">
+  <div class="wrapper">
+    <section class="footer-top">
+      <?php print render($page['footer_top']); ?>
+    </section>
+    <section class="footer-bottom">
+      <?php print render($page['footer_bottom']); ?>
```

Comment:
Good to know about the crest, we were almost going to follow the DTA.

Our current solution is to implement the inline ul in subtheme anyway. Given that the UI KIT has class `footer-links` to put the ul inline, I'm wondering if we leverage from it or not ?